MastodonSDK/Sources/MastodonSDK/Entity/Mastodon+Entity+InstanceV2.swift

```diff
@@ -0,0 +1,158 @@
+        
+        // https://github.com/mastodon/mastodon/pull/16485
+        public let configuration: Configuration?
+
+        public init(domain: String, approvalRequired: Bool? = nil) {
+            self.domain = domain
+            self.title = domain
+            self.description = ""
+            self.shortDescription = nil
+            self.email = ""
+            self.version = nil
+            self.languages = nil
+            self.registrations = nil
+            self.approvalRequired = approvalRequired
+            self.invitesEnabled = nil
+            self.urls = nil
+            self.statistics = nil
+            self.thumbnail = nil
+            self.contactAccount = nil
+            self.rules = nil
+            self.configuration = nil
+        }
+
+        enum CodingKeys: String, CodingKey {
+            case domain
+            case title
+            case description
+            case shortDescription = "short_description"
+            case email
+            case version
+            case languages
+            case registrations
+            case approvalRequired = "approval_required"
+            case invitesEnabled = "invites_enabled"
+            case urls
+            case statistics = "stats"
+            
+            case thumbnail
+            case contactAccount = "contact_account"
+            case rules
+            
+            case configuration
+        }
+    }
+}
+
+extension Mastodon.Entity.V2.Instance {
+    public struct Configuration: Codable {
+        public let statuses: Mastodon.Entity.Instance.Configuration.Statuses?
+        public let mediaAttachments: Mastodon.Entity.Instance.Configuration.MediaAttachments?
+        public let polls: Mastodon.Entity.Instance.Configuration.Polls?
+        public let translation: Mastodon.Entity.V2.Instance.Configuration.Translation?
+    
+        enum CodingKeys: String, CodingKey {
+            case statuses
+            case mediaAttachments = "media_attachments"
+            case polls
+            case translation
+        }
+    }
+}
+
+extension Mastodon.Entity.V2.Instance {
+    public struct Registrations: Codable {
+        public let enabled: Bool
+    }
+}
+
+extension Mastodon.Entity.V2.Instance.Configuration {
+    public struct Translation: Codable {
+        public let enabled: Bool
+    }
+}
+
+extension Mastodon.Entity.V2.Instance {
+    public struct Thumbnail: Codable {
+        public let url: String?
+    }
+}
+
+//extension Mastodon.Entity.V2.Instance {
+//    public struct Statuses: Codable {
+//        public let maxCharacters: Int
+//        public let maxMediaAttachments: Int
+//        public let charactersReservedPerURL: Int
+//
+//        enum CodingKeys: String, CodingKey {
+//            case maxCharacters = "max_characters"
+//            case maxMediaAttachments = "max_media_attachments"
+//            case charactersReservedPerURL = "characters_reserved_per_url"
+//        }
+//    }
```

Comment:
What's the reason you want to keep this?

Reply:
Whoopsie! Sorry I completely forgot to remove this.